isisdaeApp/Db/ADisisdae.template

```diff
@@ -326,6 +326,18 @@ record(ai, "$(P)$(R)INTG:SPEC:RATE")
     info(alarm, "ISISDAE_01")
 }
 
+record(ai, "$(P)$(R)INTG:RATE:RBV")
```

Comment:
Can we just make this and SPEC:RATE:RBV an alias of their input?

Reply:
@Adam-Szw Has this discussion resolved?

Reply:
@daryakoskeroglu Didn't see this, thank you. It's not done yet and will need a quick rework

Reply:
I am guessing this record is being created because INTG:RATE is I/O Intr and you need a non-io/intr one for a test? In which case an alias will not help as it will inherit the same SCAN. Could you instead add a `SSCN` field and then set the `INTG:RATE` into simulation mode?

Reply:
It will not work since we must be able to caput to the record in the test (which we can't do on an I/O intr record). These two record don't _really_ even need INP field since they are there to be set by the test. I tried making it an alias but then it simply wouldnt work as you then can't set value for those records.

Reply:
I've added a `SSCN` field to `INTG:RATE` - when the record is put into simulation mode you can set its value via a separate PV - see #72    This will allow you to remove the  `IFLOADTST` code from ISISComputingGroup/EPICS-ioc#653